ui/src/Logout.tsx

```diff
@@ -15,16 +13,6 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 
 const Logout: FC<WithRouterProps> = ({router}) => {
   const handleSignOut = async () => {
-    const config = await getAuth0Config()
-    if (CLOUD && config.socialSignUpOn) {
-      const auth0 = new auth0js.WebAuth({
-        domain: config.domain,
-        clientID: config.clientID,
-      })
-      auth0.logout({})
-      window.location.href = `${CLOUD_URL}${CLOUD_LOGOUT_PATH}`
-      return
-    }
```

Comment:
TLDR; we refactored cloud authentication to move away from the UI and to the API. Since the existing CLOUD API implementation for logging out already destroys Auth0 sessions & CLOUD API sessions, we can confidently depend upon the existing code and no longer need a special feature-flag enabled logout setting